ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -121,6 +121,10 @@ - (void)setImageCaptureStep:(ORKImageCaptureStep *)imageCaptureStep {
     
     _previewView.templateImage = imageCaptureStep.templateImage;
     _previewView.templateImageInsets = imageCaptureStep.templateImageInsets;
+    _previewView.accessibilityHint = imageCaptureStep.accessibilityInstructions;
+    
```

Comment:
This sets the `accessibilityInstructions` on the previewView even after the image has been captured. Maybe we want to modify `updateAppearances` to not have the accessibility instructions when there is a `capturedImage'?